### DISCUSSION TITLE
Is there a way to view all my Scrypted streams on a web page?

### DISCUSSION THREAD

**Discussion by thatdaveguy1** [koush/scrypted#425] [Ideas]
I love the Wyze Bridge web portal. It allows me to view a configurable web page, so I set up my cams with 5 second snapshots. That way I don’t have to deal with buffering video. 

But two of my cams aren’t Wyze, so I’m limited here. It would be great if Scrypted had this ability!

**Comment by koush** [comment]
https://demo.scrypted.app/#/demo
nvr paid product

**Comment by thatdaveguy1** [comment]
Wow that looks good. Still building it, or available to buy now?

**Comment by koush** [comment]
available. download the nvr plugin.

**Comment by koush** [comment]
it's beta. but fully functional. join discord, active community, and can get immediate answers to questions there.
